Library/OptimizelyClientType.swift

```diff
@@ -7,7 +7,7 @@ public protocol OptimizelyClientType: AnyObject {
   func getVariationKey(experimentKey: String, userId: String, attributes: [String: Any?]?) throws -> String
   func allExperiments() -> [String]
   func isFeatureEnabled(featureKey: String, userId: String, attributes: [String: Any?]?) -> Bool
-  func track(eventKey: String, userId: String, attributes: [String: Any?]?, eventTags: [String: Any]?) throws
```

Comment:
I think you need to keep this function signature in here because it needs to match the tracking function in the actual `Optimizely` SDK.

Reply:
I think `func track(eventName: String)` will work since the implementation of this function will go to the tracking function in the actual sdk

Reply:
Added the original track function back